src/AzureAppConfigurationImpl.ts

```diff
@@ -109,12 +109,23 @@ export class AzureAppConfigurationImpl extends Map<string, unknown> implements A
     }
 }
 
-function getValidSelectors(selectors?: { keyFilter: string, labelFilter?: string }[]) {
+function getValidSelectors(selectors?: SettingSelector[]) {
     if (!selectors || selectors.length === 0) {
         // Default selector: key: *, label: \0
         return [{ keyFilter: KeyFilter.Any, labelFilter: LabelFilter.Null }];
     }
-    return selectors.map(selectorCandidate => {
+
+    // below code dedupes selectors by keyFilter and labelFilter, the latter selector wins
+    const dedupedSelectors: SettingSelector[] = [];
+    const reversedSelectors = [...selectors].reverse();

```

Comment:
Is there a reason for choosing list reversal logic here? Since we have the same code in .NET, how about keeping the logic consistent?

Reply:
No specific reason, just for convenience, as reverse() is in-place. I'll use keep the logic consistent with .NET.